app/config/parameters.yml.dist

```diff
@@ -36,9 +36,6 @@ parameters:
     sylius.currency_importer.open_exchange_rates: EDITME
     sylius.currency_importer.ecb.base_currency: EUR
 
-    paypal.express_checkout.username: EDITME
```

Comment:
This way the application may crash, because there will be missing parameters in `parameters.yml`. It would be nice to figure out another solution for that.